### ISSUE TITLE
Race__c-Race Layout: duplicate value found: <unknown> duplicates value on record with id: <unknown>

### ISSUE THREAD

**Issue by jrattanpal** [afawcett/forcedotcom-enterprise-architecture-second-edition#1]
Created brand new DE org, ran the deploy command and it failed with this error:

`Race__c-Race Layout	Page Layout	0	0	duplicate value found: <unknown> duplicates value on record with id: <unknown>`

Am I doing anything wrong or need to prepare something? Book doesn't mention this.  I am using Spring'17 org.

**Comment by jrattanpal** [comment]
Ah, found this. https://success.salesforce.com/issues_view?id=a1p30000000T5hTAAS

**Comment by afawcett** [comment]
Ah thanks for sharing @jrattanpal, did the workaround stated on the page get you going?

**Comment by jrattanpal** [comment]
@afawcett 

I haven't tried the workaround because I haven't played with build.xml. I can try that but I might just push it manually to save time.

If I do try build.xml and find the fix then I can let you know and you can change it.

**Comment by krishNadara** [comment]
@afawcett 
Hi Andrew
I have exactly the same problem. FIrst i had an error on quick action Log a call. But when i create a Quick action Log a call, before the Ant run, it says Deployment Partially succeded, so for now it is OK.
But now am facing the problem mentionned in this ticket. I don't know how i can separate the transaction, because there is dependencies between objects, and the history tracking is turned off. 
Is there any fix? I bought the book in Amazon, so am exited to begin. 

thank you

**Comment by afawcett** [comment]
@krishNadara Sorry for the delay, have you managed to get going? If not, i can try to help.

**Comment by knadarassin** [comment]
@afawcett  no, i just put it on standby. I even tried to create a fresh new org to install the code, but same error. Actually do I need to install that to follow the exercises?

**Comment by krishNadara** [comment]
@afawcett  I was finally able to do a succcessful deployment of the code, by removing the Race object layout from the src.
So to deploy for now the steps are 
- Go to global actions, create a Global action, with action type Create a record, on the Target object  Case.
- Delete from the source the layout with the name Race__c-Race Layout.layout

**Comment by afawcett** [comment]
Thanks for sharing this @krishNadara , i need to look into this with Salesforce, it looks like a bug in the Metadata API to me.
